mentoring/templates/html/mentoring.html

```diff
@@ -1,16 +1,44 @@
-<div class="mentoring">
+<div class="mentoring" data-mode="{{ self.mode }}" data-step="{{ self.step }}">
   <div class="missing-dependency warning" data-missing="{{ self.has_missing_dependency }}">
     You need to complete <a href="{{ missing_dependency_url }}">the previous step</a> before
     attempting this step.
   </div>
-  {% for name, c in named_children %}
-  {{c.body_html|safe}}
-  {% endfor %}
-  {% if self.display_submit %}
-  <input type="button" value="submit" class="submit"></input>
-  <span class="progress" data-completed="{{ self.completed }}">
-    <span class='indicator'></span>
-  </span>
+
+  {% if title %}
+  <div class="title">
+    {% if title %} <h2>{{ title }}</h2> {% endif %}
+  </div>
   {% endif %}
-  <div class="messages"></div>
+
+  <div class="{{self.mode}}-question-block">
+    {{child_content|safe}}
+
+    {% if self.display_submit %}
```

Comment:
Update: This was a strange bug. Fixed with [this one line change](https://github.com/open-craft/xblock-mentoring/commit/4788bc70f7986b34dc79ca18dea7016ffb363c04#diff-45cffed176fc57ac8b334a66df4bebbbL144)